OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -250,12 +272,22 @@ public boolean isAnswersEnabled() {
             return mAnswersEnabled;
         }
 
-        public String getFabricKey() {
-            return mFabricKey;
+        public Kit[] getEnabledKits()   {
+            List<Kit> fabricKits = new ArrayList<>();
+
+            if (isCrashlyticsEnabled())    {
+                fabricKits.add(new CrashlyticsCore());
+            }
+
+            if (isAnswersEnabled()) {
+                fabricKits.add(new Answers());
+            }
+
+            return fabricKits.toArray(new Kit[fabricKits.size()]);
         }
 
-        public String getFabricBuildSecret() {
-            return mFabricBuildSecret;
+        public boolean isEnabled()  {
```

Comment:
Lemme merge this PR in my branch and fix this nit. And ultimately merge my branch too 🎉